include/megaapi.h

```diff
@@ -6238,6 +6240,25 @@ class MegaApi
         void setUploadLimit(int bpslimit);
 
         /**
+         * @brief Set the maximum number of connections per transfer
+         *
+         * The maximum number of allowed connections is 6. If a higher number of connections is
+         * passed to this function, it will fail with the error code API_ETOOMANY.
+         *
+         * The associated request type with this request is MegaRequest::TYPE_SET_MAX_CONNECTIONS
+         * Valid data in the MegaRequest object received on callbacks:
+         * - MegaRequest::getParamType - Returns the first parameter
+         * - MegaRequest::getNumber - Returns the second parameter
+         *
+         * @param direction Direction of transfers
+         * Valid values for this parameter are:
+         * - MegaTransfer::TYPE_DOWNLOAD = 0
+         * - MegaTransfer::TYPE_UPLOAD = 1
```

Comment:
The implementation allows to set the number of connections for both directions at once by using `direction = -1`.
We can add a new prototype, like for `pauseTranfers()`, to apply the setting to both download and upload.